### ISSUE TITLE
Upgrade Google Services Gradle plugin to `4.3.13`

### ISSUE THREAD

**Issue by ArnyminerZ** [Escalar-Alcoia-i-Comtat/Android#820]
Currently upgrading Google Service's Gradle plugin to `4.3.12` is impossible because of google/play-services-plugins#232.

Error log:
```
* What went wrong:
Execution failed for task ':base:uploadCrashlyticsMappingFileRelease'.
> Failed to calculate the value of task ':base:uploadCrashlyticsMappingFileRelease' property 'googleServicesResourceRoot'.
   > Could not get unknown property 'intermediateDir' for task ':base:processReleaseGoogleServices' of type com.google.gms.googleservices.GoogleServicesTask.
```

**Comment by ArnyminerZ** [comment]
Fixed on `4.3.13`. Merge #821

**Comment by ArnyminerZ** [comment]
Closed by #821
